kernel/arch/dreamcast/include/dc/biosfont.h

```diff
@@ -26,267 +31,296 @@
+
+/** \brief Start of DC Specific Characters in Font Block */
+#define BFONT_DREAMCAST_SPECIFIC    BFONT_WIDE_START + (7056 * BFONT_WIDE_WIDTH * BFONT_HEIGHT / 8) 
+/** \brief Takes a DC-specific icon index and returns a character offset. */
+#define BFONT_DC_ICON(offset)       BFONT_DREAMCAST_SPECIFIC + ( (offset) * BFONT_WIDE_WIDTH * BFONT_HEIGHT / 8)
+
+/** \defgroup bfont_dc_indices Dreamcast-Specific 
+    \brief    Dreamcast-specific BIOS icon offsets.
+    @{
+*/
+#define BFONT_CIRCLECOPYRIGHT       BFONT_DC_ICON(0)    /**< \brief Circle copyright */
+#define BFONT_CIRCLER               BFONT_DC_ICON(1)    /**< \brief Circle restricted */
+#define BFONT_TRADEMARK             BFONT_DC_ICON(2)    /**< \brief Trademark */
+#define BFONT_UPARROW               BFONT_DC_ICON(3)    /**< \brief Up arrow */
+#define BFONT_DOWNARROW             BFONT_DC_ICON(4)    /**< \brief Down arrow */
+#define BFONT_LEFTARROW             BFONT_DC_ICON(5)    /**< \brief Left arrow */
+#define BFONT_RIGHTARROW            BFONT_DC_ICON(6)    /**< \brief Right arrow */
+#define BFONT_UPRIGHTARROW          BFONT_DC_ICON(7)    /**< \brief Up right arrow */
+#define BFONT_DOWNRIGHTARROW        BFONT_DC_ICON(8)    /**< \brief Down right arrow */
+#define BFONT_DOWNLEFTARROW         BFONT_DC_ICON(9)    /**< \brief Down left arrow */
+#define BFONT_UPLEFTARROW           BFONT_DC_ICON(10)   /**< \brief Up left arrow */
+#define BFONT_ABUTTON               BFONT_DC_ICON(11)   /**< \brief A button */
+#define BFONT_BBUTTON               BFONT_DC_ICON(12)   /**< \brief B button */
+#define BFONT_CBUTTON               BFONT_DC_ICON(13)   /**< \brief C button */
+#define BFONT_DBUTTON               BFONT_DC_ICON(14)   /**< \brief D button */
+#define BFONT_XBUTTON               BFONT_DC_ICON(15)   /**< \brief X button */
+#define BFONT_YBUTTON               BFONT_DC_ICON(16)   /**< \brief Y button */
+#define BFONT_ZBUTTON               BFONT_DC_ICON(17)   /**< \brief Z button */
+#define BFONT_LTRIGGER              BFONT_DC_ICON(18)   /**< \brief L trigger */
+#define BFONT_RTRIGGER              BFONT_DC_ICON(19)   /**< \brief R trigger */
+#define BFONT_STARTBUTTON           BFONT_DC_ICON(20)   /**< \brief Start button */
+#define BFONT_VMUICON               BFONT_DC_ICON(21)   /**< \brief VMU icon */
+/** @} */
 
 #define BFONT_ICON_DIMEN                 32    /**< \brief Dimension of vmu icons */
 #define BFONT_VMU_DREAMCAST_SPECIFIC     BFONT_DREAMCAST_SPECIFIC+(22*BFONT_WIDE_WIDTH*BFONT_HEIGHT/8)
 /** @} */
 
-/** \defgroup vmu_icons Builtin VMU Icons
+/** \brief Builtin VMU Icons
     \ingroup  bfont_indicies
 
     Builtin VMU volume user icons. The Dreamcast's
     BIOS allows the user to set these when formatting the VMU.
+*/
+typedef enum bfont_vmu_icon {
+    BFONT_ICON_INVALID_VMU     = 0x00, /**< \brief Invalid */
+    BFONT_ICON_HOURGLASS_ONE   = 0x01, /**< \brief Hourglass 1 */
```

Comment:
Do these briefs make it much more readable in the generated docs or something? Cause they seem fairly redundant to me. If any of the define names were ambiguous at all, then maybe, but it's almost exactly just the text at `BFONT_ICON_*`